pkg/torcx/types.go

```diff
@@ -33,6 +33,8 @@ const (
 	DefaultTagRef = "com.coreos.cl"
 	// DEFAULT_PROFILE is the default profile name used
 	DEFAULT_PROFILE_NAME = "vendor"
+	// SYSTEMD_DIR is the directoy where we'll install all systemd files
+	SYSTEMD_DIR = "/run/systemd"
```

Comment:
You are totally right, and it looks like so far I always confused `transient` with `runtime`. I've updated the original ticket.